### PR TITLE
MCS-1726 - added timeout

### PR DESCRIPTION
**Pull request by NDCACI**
First pass, seemed too simple so I feel like I missed something. No test added yet.

### CODE REVIEW

**Review by ThomasSchellenbergNextCentury — CHANGES_REQUESTED** [review]
Oof, OK. My fault, for not realizing there are actually two different timeouts in the code:
- A timeout for each action step, which is set by CONFIG_TIMEOUT, which you've modified here (see `_check_for_step_timeout` in `controller.py`)
- A timeout for initializing the controller, which was the one we wanted to be able to set for this ticket (see `TIME_LIMIT_SECONDS` and `get_controller_with_timeout` in `__init__.py`)

I'm thinking it's important to keep these settings separate (at least for now), so, in order to focus on the second timeout, if you could please:
1. Add a new config setting to `config_manager.py` (maybe `CONFIG_CONTROLLER_TIMEOUT = 'controller_timeout'` ?), with a corresponding getter (you can use `180` as the default value)
2. In `__init__.py`, in `get_controller_with_timeout`, use the configured "controller_timeout" instead of TIME_LIMIT_SECONDS
3. Add new unit tests
4. Add a new section for the "controller_timeout" to “Config File Properties” in `docs/source/install.rst`

You knew this was too easy! :)

File: machine_common_sense/__init__.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def get_controller_with_timeout(unity_exec: str, config: ConfigManager):
     with concurrent.futures.ThreadPoolExecutor(max_workers=2) as executor:
         future = executor.submit(get_controller, unity_exec, config)
         try:
-            controller = future.result(timeout=TIME_LIMIT_SECONDS)
+            controller = future.result(timeout=config.get_controller_timeout)
```
Comment (ThomasSchellenbergNextCentury) [inline machine_common_sense/__init__.py:51]: You should remove the TIME_LIMIT_SECONDS variable.

Comment (ThomasSchellenbergNextCentury) [inline machine_common_sense/__init__.py:47]: Need parentheses after the function call (`config.get_controller_timeout()`)

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]
Looks great!

**Review by jaudick — APPROVED** [review]

**Review by brianpippin — APPROVED** [review]

**Pull request merged**